flocker/node/agents/blockdevice.py

```diff
@@ -869,12 +878,12 @@ def run(self, deployer):
         )
         device = api.get_device_path(volume.blockdevice_id)
 
-        # This duplicates CreateFilesystem now.
-        check_output(["mkfs", "-t", "ext4", device.path])
+        create = CreateFilesystem(volume=volume, filesystem=u"ext4")
```

Comment:
Without the logging assertion change, the only failures I see are in `flocker.node.agents.test.test_blockdevice.ResizeBlockDeviceDatasetTests` which it seems shouldn't be affected by `CreateBlockDeviceDataset` changes.